dags.yaml

```diff
@@ -734,3 +734,20 @@ bqetl_acoustic_raw_recipient_export:
     retry_delay: 5m
   tags:
     - impact/tier_3
+
+bqetl_analytics_aggregations:
+  default_args:
+    depends_on_past: false
+    email: ["telemetry-alerts@mozilla.com", "lvargas@mozilla.com"]
```

Comment:
should all of data engineering get an email about failures related to this? @gkabbz I wonder, would it make sense for us to get a team mailing list set up and use for cases like this?

Reply:
I'd say the aero team in addittion to telemetry-alerts.
I set it as tier 1 to get a ticket created if it fails as data will be unavailable.

Reply:
I'm still not sure if all of data engineering should get emailed about this, whoever is on Airflow triage should create a bug for this if it fails. If we all get notified it should be enough IMHO, it will be one of us fixing it anyways.

I'm just very skeptical about everyone being notified about everything, what tends to happen is that people start treating these emails as noise and stops paying attention to those emails.

Reply:
^^ But that's only my opinion and the decision is up to you.

Reply:
Added George for now, for a second notification within the team.